architectures/decentralized/solana-treasurer/programs/solana-treasurer/src/logic/participant_claim.rs

```diff
@@ -12,9 +12,6 @@ use crate::ProgramError;
 #[derive(Accounts)]
 #[instruction(params: ParticipantClaimParams)]
 pub struct ParticipantClaimAccounts<'info> {
-    #[account(mut)]
-    pub payer: Signer<'info>,
-
```

Comment:
original feedback:
----------

In two instructions, namely run_top_up and participant_claim a payer (that has to be a signer) is a required account, however that account is never used in the function.

```rust
#[account(mut)]
pub payer: Signer<'info>,
```